compiler/AST/stmt.cpp

```diff
@@ -600,6 +600,58 @@ BlockStmt::isCForLoop() const {
   return false;
 }
 
+static CallExpr* getMarker(BlockStmt* stmt, PrimitiveTag markerType) {
```

Comment:
I have a:

- strong preference for making this a private class function instead of a static helper
- milder preference for renaming it to `getMarkerPrimIfExists`